### ISSUE TITLE
Baseline models

### ISSUE THREAD

**Issue by julianikulski** [julianikulski/data-science-resources#6]
**Resource**: https://machinelearningmastery.com/naive-classifiers-imbalanced-classification-metrics/
**Description**: What Is the Naive Classifier for Each Imbalanced Classification Metric?

**Resource**: https://scikit-learn.org/stable/modules/model_evaluation.html#dummy-estimators
**Description**: 3.3.6. Dummy estimators

**Resource**: https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RepeatedStratifiedKFold.html#
**Description**: sklearn.model_selection.RepeatedStratifiedKFold
